CHANGELOG.md

```diff
@@ -7,15 +7,14 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/compare/v0.6.0...HEAD)
 
-- Added a fallback service name if one is not
-provided
-- The fallback service name is based on the
-Assembly name or ProcessName, if the application is .NET Framework and hosted
-in IIS this will be the SiteName/ApplicationVirtualPath.
-
 ### Added
 
+- Stability status into documentation.
 - Support `OTEL_LOG_LEVEL` to configure SDK logging level.
+- Add a fallback service name if one is not provided.
+  The fallback service name is based on the
+  Assembly name or ProcessName, if the application is .NET Framework and hosted
+  in IIS this will be the SiteName/ApplicationVirtualPath.
 
```

Comment:
```suggestion
- Stability status added to the documentation.
- Support `OTEL_LOG_LEVEL` to configure SDK logging level.
- Add a fallback service name if none is provided.
  The fallback service name is based on the name of either
the assembly or the process. If the application uses .NET Framework and is hosted
  on IIS, the service name is determined using  `SiteName/ApplicationVirtualPath`.
```

`AssemblyName` and `ProcessName` if using the objects as such. If not, just "assembly name and process name.

Which comes first if both are available, by the way? Assembly or process? You might want to clarify that.

Reply:
In order:
 - Env. variable
 - IIS
 - AssemblyName
 - Process Name

Reply:
Assembly name comes first.